### PR TITLE
Changed functionality of the indexer in findObjects

### PR DESCRIPTION
**Pull request by RobertPoienar**

### CODE REVIEW

**Comment by RobertPoienar** [comment]
@robert-altom @gomboskriszta15 @ReichEdvin @rucindrea

File: Docs/source/pages/commands.md [hunk 1]
```diff
@@ -26,6 +26,9 @@ Once you have an instance of the _AltDriver_, you can use all the available comm
 
 Finds the first object in the scene that respects the given criteria. Check [By](#by-selector) for more information about criteria.
 
+.. important::
+     Indexer functionality was changed in 2.0.2 to match that of XPath and it no longer return the n-th child of the object. Now it returns the n-th object that respects the selectors (Name, Component, Tag, etc.) from the objects with the same parent. For example //Button/Text[1] will return the second object named `Text` that is the child of the `Button`
```
Comment (gomboskriszta15) [inline Docs/source/pages/commands.md:30]:
```suggestion
     Indexer functionality was changed in 2.0.2 to match that of XPath and it no longer returns the n-th child of the object. Now it returns the n-th object that respects the selectors (Name, Component, Tag, etc.) from the objects with the same parent. For example //Button/Text[1] will return the second object named `Text` that is the child of the `Button`
```

File: Docs/source/pages/commands.md [hunk 2]
```diff
@@ -80,6 +83,9 @@ Finds the first object in the scene that respects the given criteria. Check [By]
 
 Finds all objects in the scene that respects the given criteria. Check [By](#by-selector) for more information about criteria.
 
+.. important::
+     Indexer functionality was changed in 2.0.2 to match that of XPath and it no longer return the n-th child of the object. Now it returns the n-th object that respects the selectors (Name, Component, Tag, etc.) from the objects with the same parent. For example //Button/Text[1] will return the second object named `Text` that is the child of the `Button`
```
Comment (gomboskriszta15) [inline Docs/source/pages/commands.md:87]:
```suggestion
     Indexer functionality was changed in 2.0.2 to match that of XPath and it no longer returns the n-th child of the object. Now it returns the n-th object that respects the selectors (Name, Component, Tag, etc.) from the objects with the same parent. For example //Button/Text[1] will return the second object named `Text` that is the child of the `Button`
```

File: Docs/source/pages/commands.md [hunk 3]
```diff
@@ -138,6 +144,9 @@ Finds all objects in the scene that respects the given criteria. Check [By](#by-
 
 Finds the first object in the scene that respects the given criteria. Check [By](#by-selector) for more information about criteria.
 
+.. important::
+     Indexer functionality was changed in 2.0.2 to match that of XPath and it no longer return the n-th child of the object. Now it returns the n-th object that respects the selectors (Name, Component, Tag, etc.) from the objects with the same parent. For example //Button/Text[1] will return the second object named `Text` that is the child of the `Button`
```
Comment (gomboskriszta15) [inline Docs/source/pages/commands.md:148]:
```suggestion
     Indexer functionality was changed in 2.0.2 to match that of XPath and it no longer returns the n-th child of the object. Now it returns the n-th object that respects the selectors (Name, Component, Tag, etc.) from the objects with the same parent. For example //Button/Text[1] will return the second object named `Text` that is the child of the `Button`
```

**Review by rucindrea — APPROVED** [review]

**Review by gomboskriszta15 — APPROVED** [review]

**Pull request merged**